### PR TITLE
README: Add related tools

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -775,6 +775,10 @@ A [GitHub Action](https://github.com/oss-review-toolkit/ort-ci-github-action) to
 
 A [GitLab Pipeline](https://github.com/oss-review-toolkit/ort-gitlab-ci) to run ORT for your GitLab repositories.
 
+## ORT Config Repository
```
Comment (sschuberth) [inline README.md:778]: I guess this should go first, as it's "closest" to ORT core as a "tool".

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ use):
 * [_Notifier_](./notifier) - sends result notifications via different channels (like
   [emails](./examples/notifications/src/main/resources/example.notifications.kts) and / or JIRA tickets).
 
+Also see the [list of related tools](#related-tools) that help with running ORT.
```
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply README.md:58]: Any specific idea? Maybe rename the Wiki section to "Alternative tools"?

[the remark this answers is not in the document]
Comment (sschuberth) [reply README.md:58]: Maybe use "associated tools" or "accompanying tools" here, or "similar tools" in the wiki?

[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply README.md:58]: I would prefer the latter.

[the remark this answers is not in the document]
Comment (sschuberth) [reply README.md:58]: I actually also like "accompanying tools" for here, as it sound more "close" than just "related". But feel free to rename the wiki page, that's also fine with me. We could also do both renamings, and use "related" in neither place. Your call.

**Review by sschuberth — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply README.md:58]: I have renamed the wiki page. I don't like "accompanying" because I can't even pronounce it as a non-native speaker ;)

**Comment by mnonnenmacher** [comment]
Merged because failed link check is unrelated to the change and the links do actually work.

**Pull request merged**